simpleLooping/isPrime.py

```diff
@@ -3,3 +3,11 @@
 # Ex:
 #   isPrime(937453153313) # return True
 #   isPrime(965246846246) # return False
+def isPrime(number: int) -> bool:
+    if number < 2:
+        return False
+    for divisor in range(2, int(number ** 0.5) + 1):
```

Comment:
is the conditional structure and the loop structure part of the same code block? I thought one is the "guarding condition code block" and the other is the "prime checking code block"?